app/lib/plan_rule_loader.rb

```diff
@@ -19,20 +19,20 @@ module PlanRuleLoader
         web_hooks
         iam_tools
       ],
-      'metadata' => { 'cannot_automatically_be_upgraded_to' => true }
+      metadata: { cannot_automatically_be_upgraded_to: true }
     }
   }.freeze
 
   module_function
 
   def load_config
-    config = (ThreeScale.config.plan_rules || {}).reverse_merge DEFAULT_RULES
+    config = (ThreeScale.config.plan_rules.deep_symbolize_keys || {}).reverse_merge DEFAULT_RULES
```

Comment:
Is this needed? I understand that now all configs pulled in by `config_for` are already deep-symbolized.

Reply:
I've been trying and it seems it's actually required. For instance, [this call](https://github.com/3scale/porta/blob/THREESCALE-7405-rails-6/config/application.rb#L243) to `config_for` returns string keys. And [the call](https://github.com/3scale/porta/blob/THREESCALE-7405-rails-6/config/application.rb#L228) that reads the plan rules uses `try_config_for` which calls `symbolize_keys`, but not `deep_*` so only the top level keys are symbolized.

Reply:
Hmmm, I don't get it... 

Accessing config using non-symbol keys is deprecated, but the keys are not symbolized automatically?... :thinking: 

Let me double-check what's going on there

Reply:
https://github.com/3scale/porta/pull/3501

Up to you if you want to merge this or not :) 

I just felt curious about this change.

Also, if I am wrong, please let me know.